src/managers/GuildApplicationCommandManager.js

```diff
@@ -119,6 +120,119 @@ class GuildApplicationCommandManager extends ApplicationCommandManager {
+   * to remove from the command permissions
+   * @returns {Promise<ApplicationCommandPermissions[]>}
+   * @example
+   * // Remove a user permission from this command
+   * guild.commands.removePermissions('123456789012345678', '876543210123456789')
+   *   .then(console.log)
+   *   .catch(console.error);
+   * @example
+   * // Remove multiple roles from this command
+   * guild.commands.removePermissions('123456789012345678', [ '876543210123456789', '765432101234567890' ])
+   *    .then(console.log)
+   *    .catch(console.error);
+   */
+  async removePermissions(command, userOrRole) {
+    const id = this.resolveID(command);
+    if (!id) throw new TypeError('INVALID_TYPE', 'command', 'ApplicationCommandResolvable');
+
+    let permissions = await this.fetchPermissions(id).catch(err => {
+      if (err.httpStatus !== 404) throw err;
+      return [];
+    });
+
+    if (Array.isArray(userOrRole)) {
+      const ids = userOrRole.map(x => {
+        const resolved = this.client.users.resolveID(x) ?? this.guild.roles.resolveID(x);
+        if (resolved) return resolved;
+        throw new TypeError('INVALID_ELEMENT', 'Array', 'userOrRole', x);
+      });
+
+      permissions = permissions.filter(perm => !ids.includes(perm.id));
+    } else {
+      if (typeof userOrRole !== 'string') {
+        userOrRole = this.client.users.resolveID(userOrRole) ?? this.guild.roles.resolveID(userOrRole);
+        if (!userOrRole) {
+          throw new TypeError('INVALID_TYPE', 'userOrRole', 'UserResolvable or RoleResolvable');
+        }
+      }
+      permissions = permissions.filter(perm => perm.id !== userOrRole);
+    }
+
+    return this.setPermissions(id, permissions);
+  }
+
+  /**
+   * Check whether a command can be used by a user or role
+   * @param {ApplicationCommandResolvable} command The command to verify the permissions
+   * @param {UserResolvable|RoleResolvable} userOrRole The user or role that you want to check if has permission to use
+   * this command.
+   * @returns {Promise<boolean>}
+   * @example
+   * // Check whether a user has permission to use a command
+   * guild.commands.hasPermission('123456789012345678', '876543210123456789')
+   *  .then(console.log)
+   *  .catch(console.error);
+   */
+  async hasPermission(command, userOrRole) {
+    const id = this.resolveID(command);
+    if (!id) throw new TypeError('INVALID_TYPE', 'command', 'ApplicationCommandResolvable');
+
+    const permissions = await this.fetchPermissions(id).catch(err => {
+      if (err.httpStatus !== 404) throw err;
+      return [];
+    });
+
+    if (typeof userOrRole !== 'string') {
+      userOrRole = this.client.users.resolveID(userOrRole) ?? this.guild.roles.resolveID(userOrRole);
+      if (!userOrRole) {
+        throw new TypeError('INVALID_TYPE', 'userOrRole', 'UserResolvable or RoleResolvable');
+      }
+    }
+    return !!permissions.find(perm => perm.id === userOrRole)?.permission;
```

Comment:
I believe this should not return the permission, instead returning simply whether or not the id exists in the current permission overwrites. Otherwise how could you tell the difference between the permission not being set (thus true) and it being set false when `default_permission: true`